portway/portway
# Feature per user billing

- adds per user plan (id is via env STRIPE_PER_USER_PLAN_ID)
- new test setup file for API, which allows us to pass in test specific ENV
- removes plan selector from client
- new users get PER_USER plan
- if a SINGLE_USER adds seats, they get upgraded to PER_USER plan

To test:
on current release, make 3 accounts, and go through the steps to make:
1. a SINGLE_USER with billing info(TRIALING_PENDING_ACTIVE)
2. a MULTI_USER with billing info
3. keep one as a TRIALING account

change to this branch and test by adding billing info/adding seats for each user, and verify that their organization billing info and ORG get the new plan type

Then - create a user on the feature branch and go through each billing option
